Apps/Src/ArrayConnection.c

```diff
@@ -1,7 +1,6 @@
 /* Copyright (c) 2020 UT Longhorn Racing Solar */
 
-#include "ArrayConnection.h"
-#include "Contactors.h"
+#include "Tasks.h"
```

Comment:
I'm not a big fan of putting all of the `#include` statements in Tasks.h and making everything just include Tasks.h because it hides the dependencies for each file